src/components/connections.vue

```diff
@@ -14,14 +14,39 @@ export default {
     }
   },
   methods: {
-    ...mapActions(['connect', 'setDatabase', 'setCollection']),
+    ...mapActions([
+      'connect',
+      'setDatabase',
+      'setCollection',
+      'lastOpenedCollection',
+      'addCollectionToOpenedCollections'
+    ]),
     async doConnect() {
       await this.connect(this.connectionString)
       this.$router.push('/connection')
+    },
+    setSelectedDBorCollection(sourceType = 'db', sourceName) {
```

Comment:
please see my related comment on the previous PR — this method should be in the store.